server/helpers/profiler.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @description Delete unnecessary profile
+ * @param {object} userData Data of the user
+ * @returns {object} Necessary user profile details
+ */
+const getProfile = userData => {
+  delete userData.dataValues.password;
+  delete userData.dataValues.is_activated;
+
+  return userData;
+};
+
+export default getProfile;
```

Comment:
You don't need this helper function to delete these values. Use the options I showed you on the call